test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -187,9 +187,14 @@ public void Func([Queue(QueueName)] out string x)
         public void Fails_When_No_Storage_is_set()
         {
             var host = TestHelpers.NewJobHost<ProgramSimple>();  // no storage account!
-
+            string message = "Microsoft Azure WebJobs SDK Storage connection string is missing or empty. " +
+                       "The Microsoft Azure Storage account connection string can be set in the following ways:" + Environment.NewLine +
+                       "1. Set the connection string named 'AzureWebJobsStorage' in the connectionStrings section of the .config file in the following format " +
+                       "<add name=\"AzureWebJobsStorage\" connectionString=\"DefaultEndpointsProtocol=http|https;AccountName=NAME;AccountKey=KEY\" />, or" + Environment.NewLine +
+                       "2. Set the environment variable named 'AzureWebJobsStorage', or" + Environment.NewLine +
+                       "3. Set corresponding property of JobHostConfiguration.";
             TestHelpers.AssertIndexingError(() => host.Call("Func"),
-                "ProgramSimple.Func", "Unable to bind Queue because no storage account has been configured.");
```

Comment:
So there is now some dead code in QueueBindingProvider.ToParameterDescriptorForCollector. Recommend removing that since the account there can't be null anymore.